receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/constant.go

```diff
@@ -74,11 +74,14 @@ const (
 	AttributeStorageRead  = "storage.read_bytes"
 	AttributeStorageWrite = "storage.write_bytes"
 
+	AttributeDuration = "duration"
+
 	UnitBytes       = "Bytes"
 	UnitMegaBytes   = "Megabytes"
 	UnitNanoSecond  = "Nanoseconds"
 	UnitBytesPerSec = "Bytes/Second"
 	UnitCount       = "Count"
 	UnitVCpu        = "vCPU"
 	UnitPercent     = "Percent"
+	UnitSecond      = "Second"
```

Comment:
`Second` or `Seconds`? As we used plural version like `Bytes`, `Nanoseconds`.

Reply:
Sure, I will update it.